.travis.yml

```diff
@@ -1,8 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - '5'
-  - '4'
-  - '0.10'
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
+  - "iojs"
 before_install:
-  - npm install -g npm
+  - if [ "$TRAVIS_NODE_VERSION" = "0.10" ]; then npm install -g npm@2; fi
+matrix:
+  fast_finish: true
+cache:
+  directories:
+    - node_modules
```

Comment:
This could potentially lead to unnoticed broken builds when a subdependency breaks.